src/api/Feed.js

```diff
@@ -571,23 +573,16 @@ class Feed extends Base {
         this.errorCallback = onError;
 
         // Using the UAA File Activities endpoint replaces the need for these calls
-        const annotationsPromise =
-            !shouldUseUAA && shouldShowAnnotations
-                ? this.fetchAnnotations(permissions, shouldShowReplies)
-                : Promise.resolve();
+        const annotationsPromise = shouldShowAnnotations
+            ? this.fetchAnnotations(permissions, shouldShowReplies)
+            : Promise.resolve();
         const commentsPromise = () => {
```

Comment:
nit: commentsPromise could be updated to match the ternary pattern of the others?